docs/ref/contrib/postgres/fields.txt

```diff
@@ -245,10 +245,12 @@ lookups available after the transform do not change. For example::
 Indexing ``ArrayField``
 -----------------------
 
-At present using :attr:`~django.db.models.Field.db_index` will create a
-``btree`` index. This does not offer particularly significant help to querying.
-A more useful index is a ``GIN`` index, which you should create using a
-:class:`~django.db.migrations.operations.RunSQL` operation.
+In addition to a :class:`~django.db.models.Index` or
+:attr:`~django.db.models.Field.db_index` which will create a ``btree``
+index, ``ArrayField`` can be indexed using
+:class:`~django.contrib.postgres.indexes.GinIndex` or a ``gist`` index
```

Comment:
a gist index. The latter can be implemented with an class:`~django.db.models.Index` subclass.